doc/install/dependencies.rst

```diff
@@ -222,14 +222,15 @@ Build dependencies
 Python
 ------
 
-By default, ``pip`` will build packages using build isolation, meaning that these
-build dependencies are temporally installed by pip for the duration of the
-Matplotlib build process. However, build isolation is disabled when :ref:`installing Matplotlib for development <development-install>`;
-therefore we recommend using one of our :ref:`virtual environment configurations <dev-environment>` to
-create a development environment in which these packages are automatically installed.
-
-If you are developing Matplotlib and unable to use our environment configurations,
-then you must manually install the following packages into your development environment:
+``pip`` normally builds packages using :external+pip:doc:`build isolation <reference/build-system/pyproject-toml>`,
+which means that ``pip`` installs the dependencies listed here for the
+duration of the build process; However, build isolation is disabled when
+:ref:`installing Matplotlib for development <development-install>`.
+
+When build isolation is disabled (:external+pip:ref:`--no-build-isolation <install_--no-build-isolation>`),
```

Comment:
Alternative:

> and therefore you have to install the build dependencies yourself. This can be done by :ref:`creating a virtual environment <dev-environment>` or manually

Optionl, but slightly more connected. We don't need a separate paragraph.